modules/relaidoBidAdapter.js

```diff
@@ -104,9 +105,10 @@ function buildRequests(validBidRequests, bidderRequest) {
     uuid: getUuid(),
     pv: '$prebid.version$',
     imuid: imuid,
+    canonical_url_hash: getCanonicalUrlHash(bidderRequest.refererInfo),
```

Comment:
why do you want the url hash? why not just hash it server side?

Reply:
@patmmccann 
The reason for hashing is to reduce the computational load on the server side. It also has the advantage of data reduction since the value is a fixed length.